yarn-project/sequencer-client/src/sequencer/sequencer.ts

```diff
@@ -169,7 +169,7 @@ export class Sequencer {
       const block = await this.buildBlock(processedValidTxs, l1ToL2Messages, emptyTx, newGlobalVariables);
       this.log(`Assembled block ${block.number}`);
 
-      await this.publishExtendedContractData(validTxs, block);
+      await this.publishExtendedContractData(processedTxs, block);
```

Comment:
Shouldn't this be `processedValidTxs`..? I may be mistaken though, with so many phases going on here.